app/models/grade.rb

```diff
@@ -0,0 +1,2 @@
+class Grade < ApplicationRecord
```

Comment:
add validations
number - from 1 to 11 (or 12 maybe :))
group - capitalized character

Reply:
i have done this.